modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderItemLocalServiceImpl.java

```diff
@@ -1951,6 +1954,67 @@ private boolean _jsonMatches(String json1, String json2) {
 			return true;
 		}
 
+		try {
```

Comment:
This is super-intensive but currently necessary to handle the different formats for items with options coming from Add to Cart or Quick Add to Cart. It'll be solved elsewhere via: [LPD-25497](https://liferay.atlassian.net/browse/LPD-25497)